<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDeviceChooser</class>
 <widget class="QWidget" name="TableDeviceChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>326</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>232</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="KPushButton" name="addCustomDeviceButton"/>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="detailsButton"/>
   </item>
   <item row="1" column="2">
    <widget class="KPushButton" name="testButton"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
